### PR TITLE
Add non transactional merge doc for Ignite

### PR DESCRIPTION
**Pull request by chenjian2664**
## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.
( ) Release notes are required. Please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
## Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

**Comment by chenjian2664** [comment]
For #24443

File: docs/src/main/sphinx/connector/ignite.md [hunk 1]
```diff
@@ -78,6 +78,18 @@ configured connector to create a catalog named `sales`.
 ```{include} non-transactional-insert.fragment
 ```
 
+### Non-transactional MERGE
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/ignite.md:81]: We should merge #24523 first and include the fragment in my opinion.

Comment (chenjian2664) [reply docs/src/main/sphinx/connector/ignite.md:81]: Sure!

**Comment by mosabua** [comment]
I will get my PR ready and include Ignite so you can close this .. I will definitely do it before the next release in January.

**Pull request closed without merging**